minimesos/src/main/java/com/containersol/minimesos/mesos/MesosMasterContainer.java

```diff
@@ -79,6 +83,15 @@ protected CreateContainerCmd dockerCommand() {
     }
 
     @Override
+    public State getState() {
```

Comment:
Implementation of this method in Master and Agent seem to be identical. Can it be moved to the common parent class?